utilities.py

```diff
@@ -213,3 +249,85 @@ def best_id(ids):
         return ids['tvrage']
     elif 'slug' in ids:
         return ids['slug']
+
+
+def checkExcludePath(excludePath, excludePathEnabled, fullpath, x):
+    if excludePath != "" and excludePathEnabled and fullpath.startswith(excludePath):
+        logger.debug(
+            "checkExclusion(): Video is from location, which is currently set as excluded path %i." % x)
+        return True
```

Comment:
https://github.com/trakt/script.trakt/pull/294/files#r57535233 needs this to return `False` by default, otherwise it would return `None` and everything would explode:
```python
TypeError: unsupported operand type(s) for |=: 'bool' and 'NoneType'
```